### PR TITLE
Fix overflow scrolling of sidenav docs

### PR DESCRIPTION
**Pull request by ross-pfahler**
This is a standalone follow up from @snowystinger's comment here: https://github.com/adobe-private/react-spectrum-v3/pull/415#discussion_r409785598

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [Issue](https://github.com/adobe-private/react-spectrum-v3/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

## 📝 Test Instructions:

Sidenav of docs should scroll with no large gap under.

## 🧢 Your Team:

React Spectrum

### CODE REVIEW

File: packages/@adobe/spectrum-css-temp/components/sidenav/index.css [hunk 1]
```diff
@@ -21,8 +21,6 @@ governing permissions and limitations under the License.
   margin: 0;
   padding: 0;
   outline: none;
-
-  height: 100%;
```
Comment (dannify) [inline packages/@adobe/spectrum-css-temp/components/sidenav/index.css:25]: So whats going on here? Why do we need to change the Spectrum css?
@snowystinger

Comment (snowystinger) [reply packages/@adobe/spectrum-css-temp/components/sidenav/index.css:25]: ah, yep, i meant in the docs css

Comment (ross-pfahler) [reply packages/@adobe/spectrum-css-temp/components/sidenav/index.css]: thanks @dannify + @snowystinger -- updated.

**Comment by ross-pfahler** [comment]
covered in https://github.com/adobe-private/react-spectrum-v3/pull/415

**Pull request closed without merging**